Sources/StreamChat/WebSocketClient/Events/UserEvents.swift

```diff
@@ -4,17 +4,34 @@
 
 import Foundation
 
-public struct UserPresenceChangedEvent: UserSpecificEvent {
-    public let userId: UserId
-    public let createdAt: Date?
+/// Triggered when user status changes (eg. online, offline, away, etc.)
+public struct UserPresenceChangedEvent: Event {
+    /// The user the status changed for
+    public let user: ChatUser
     
+    /// The event timestamp
+    public let createdAt: Date?
```

Comment:
The `createdAt` must exist but it's exposed as optional so we have to keep it as it is until the next major